Assets/Mirror/Components/NetworkTransform/NetworkTransformHybrid2022.cs

```diff
@@ -0,0 +1,1145 @@
+        public int sendRate = 30; // in Hz. easier to work with as int for EMA. easier to display '30' than '0.333333333'
+        public float sendInterval => 1f / sendRate;
+        // END CUSTOM CHANGE
+
+        [Tooltip("Ocassionally send a full reliable state to delta compress against. This only applies to Components with SyncMethod=Unreliable.")]
+        public int baselineRate = 1;
+        public float baselineInterval => baselineRate < int.MaxValue ? 1f / baselineRate : 0; // for 1 Hz, that's 1000ms
+        double lastBaselineTime;
+        double lastDeltaTime;
+
+        // delta compression needs to remember 'last' to compress against.
+        // this is from reliable full state serializations, not from last
+        // unreliable delta since that isn't guaranteed to be delivered.
+        byte lastSerializedBaselineTick = 0;
+        Vector3 lastSerializedBaselinePosition = Vector3.zero;
+        Quaternion lastSerializedBaselineRotation = Quaternion.identity;
+
+        // save last deserialized baseline to delta decompress against
+        byte lastDeserializedBaselineTick = 0;
+        Vector3 lastDeserializedBaselinePosition = Vector3.zero;
+        Quaternion lastDeserializedBaselineRotation = Quaternion.identity;
+
+        // only sync when changed hack /////////////////////////////////////////
+        [Header("Sync Only If Changed")]
+        [Tooltip("When true, changes are not sent unless greater than sensitivity values below.")]
+        public bool onlySyncOnChange = true;
+
+        // change detection: we need to do this carefully in order to get it right.
+        //
+        // DONT just check changes in UpdateBaseline(). this would introduce MrG's grid issue:
+        //   server start in A1, reliable baseline sent to client
+        //   server moves to A2, unreliabe delta sent to client
+        //   server moves to A1, nothing is sent to client becuase last baseline position == position
+        //   => client wouldn't know we moved back to A1
+        //
+        // INSTEAD: every update() check for changes since baseline:
+        //   UpdateDelta() keeps sending only if changed since _baseline_
+        //   UpdateBaseline() resends if there was any change in the period since last baseline.
+        //   => this avoids the A1->A2->A1 grid issue above
+        bool changedSinceBaseline = false;
+
+        // sensitivity is for changed-detection,
+        // this is != precision, which is for quantization and delta compression.
+        [Header("Sensitivity"), Tooltip("Sensitivity of changes needed before an updated state is sent over the network")]
+        public float positionSensitivity = 0.01f;
+        public float rotationSensitivity = 0.01f;
+        // public float scaleSensitivity    = 0.01f;
+
+        [Tooltip("Enable to send all unreliable messages twice. Only useful for extremely fast paced games since it doubles bandwidth costs.")]
```

Comment:
```suggestion
        [Tooltip("Enable to send all unreliable messages twice. Only useful for extremely fast-paced games since it doubles bandwidth costs.")]
```